Example.do

```diff
@@ -62,14 +62,80 @@ projvar y x*, p(w_)
 reg w_y w_x* if sample==1, noc robust
 drop w_*
 
-twowayset hid tid, save
-twowaysample hid tid, gen(sample2) save
-projvar y x*, p(w_) save
-reg w_y w_x* if sample2==1, noc robust
+twowaysave hid tid
+}
+}
+}
+}
+
+
+********** RUN AGAIN USING TWOWAYLOAD
+clear all
+do twowayreg.ado
+
+*** 0) Preliminaries
```

Comment:
Let's not copy the code from above. Instead, you can save the stata dataset and then load it again.